src/scss/components/_start-link.scss

```diff
@@ -0,0 +1,7 @@
+@use 'microscope-sass/lib/bem';
+
+.openforms-start-link {
+  @include bem.element('icon') {
+    margin-inline-start: 10px;
```

Comment:
consider the flexbox approach here too :) `display: inline-flex` can be useful.

For a complicated example how we apply spacings, see `_app.scss`

Reply:
Mhh yeah that is an option... So that we have the flexibility to display the icon on either side?

I assume a `display: flex` (or `inline-flex` 😉 ) with a `row-gap: 10px;` would be enough, but i'll take a look